core/amber/src/main/protobuf/edu/uci/ics/amber/engine/common/actormessage.proto

```diff
@@ -21,7 +21,7 @@ message CreditUpdate {
 message ActorCommand {
   oneof sealed_value {
       Backpressure backpressure = 1;
-      CreditUpdate credit_update = 2;
```

Comment:
what's the correct format to declare it? I've seem both camelCase and snake_case...

Reply:
I will do another PR to unify the format.